tasks/test.js

```diff
@@ -1,7 +1,31 @@
 import _, { keys } from 'lodash';
+import fs from 'fs';
+import path from 'path';
+
 const visualRegression = require('../utilities/visual_regression');
 
 module.exports = function (grunt) {
+  grunt.registerTask('checkPlugins', 'Checks for plugins which may disrupt tests', function checkPlugins() {
```

Comment:
Since this is in the test task, we should namespace it to keep consistent with the rest of the tasks here. `test:checkPlugins`

_(say test task 5 times fast)_